src/main/protobuf/dwrf_proto.proto

```diff
@@ -101,13 +101,16 @@ message Stream {
     optional uint64 length = 3;
     optional bool useVInts = 4 [default = true];
     optional uint32 sequence = 6;
+    // Offset of the stream so that we don't need dummy streams for encrypted columns
+    optional uint64 offset = 8;
```

Comment:
i would like to call out that offset follows the same optimization as keys when it only need to be set a the run start. If offset of one stream can be interpreted from offset of previous stream + length, then offset doesn't have to be set